src/plone/volto/scripts/add_image_field_metadata.py

```diff
@@ -0,0 +1,24 @@
+"""
+bin/instance -O Plone run scripts/add_image_field_metadata.py
+"""
+from plone import api
+from plone.app.contenttypes.behaviors.leadimage import ILeadImageBehavior
+from plone.volto.behaviors.preview import IPreview
+from plone.volto.scripts.utils import print_info
+import transaction
+
+
+catalog = api.portal.get_tool("portal_catalog")
+
+with api.env.adopt_roles(["Manager"]):
+    images = api.content.find(object_provides=ILeadImageBehavior.__identifier__)
+    preview_images = api.content.find(object_provides=IPreview.__identifier__)
+
+    brains = images + preview_images
+
+    for brain in brains:
+        obj = brain.getObject()
+        catalog.catalog_object(obj)
```

Comment:
@sneridagh @ericof shouldn't we just reindex the new index here instead of the full object?

Reply:
I had the same thing, but It's not an index, it's metadata. So you have to reindex the whole object. I tried to investigate if there's way to update the metadata, but I couldn't find one.